recording-oracle/src/modules/campaigns/campaigns.repository.ts

```diff
@@ -32,16 +32,15 @@ export class CampaignsRepository extends Repository<CampaignEntity> {
 
     const results = await this.createQueryBuilder('campaign')
       .where('campaign.status = :status', { status: CampaignStatus.ACTIVE })
-      .andWhere('campaign.startDate <= :timeAgo', { timeAgo })
       .andWhere(
```

Comment:
Now it read as: "give me all campaigns with `active` status that either:
- ended
- started day ago but no results recorded yet
- more than a day from last results recording passed